android/src/main/java/com/expensify/reactnativekeycommand/KeyCommandModule.java

```diff
@@ -28,7 +29,7 @@ public class KeyCommandModule extends ReactContextBaseJavaModule {
 
     public static final String NAME = "KeyCommand";
 
-    private static final Set<Set<Object>> commandsArray = new HashSet<Set<Object>>();
+    private static final Set<ReadableMap> commandsArray = new HashSet<ReadableMap>();
```

Comment:
```suggestion
    private static final HashSet<ReadableMap> commandsArray = new HashSet<ReadableMap>();
```

Should this be like this?

Reply:
HashSet is an abstraction on top of set so current type is correct.

https://github.com/facebook/react-native/blob/8bd3edec88148d0ab1f225d2119435681fbbba33/ReactAndroid/src/test/java/com/facebook/react/CompositeReactPackageTest.java#L110-L112
https://github.com/facebook/react-native/blob/8bd3edec88148d0ab1f225d2119435681fbbba33/ReactAndroid/src/main/java/com/facebook/react/bridge/JavaScriptModuleRegistry.java#L59